src/components/InputFieldComponent/InputFieldComponent.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+
+function InputFieldComponent (props) {
+	return (
+		<div>
+		<label> {props.label}</label>
```

Comment:
indendation level for label is wrong. If something is a child element it should be visible thus making the code more readable